src/time_integration/methods_2N.jl

```diff
@@ -104,9 +104,8 @@ function Base.getproperty(integrator::SimpleIntegrator2N, field::Symbol)
     return getfield(integrator, field)
 end
 
-# Fakes `solve`: https://diffeq.sciml.ai/v6.8/basics/overview/#Solving-the-Problems-1
-function solve(ode::ODEProblem, alg::T;
-               dt, callback = nothing, kwargs...) where {T <: SimpleAlgorithm2N}
+function init(ode::ODEProblem, alg::T;
+              dt, callback = nothing, kwargs...) where {T <: SimpleAlgorithm2N}
```

Comment:
```suggestion
function init(ode::ODEProblem, alg::SimpleAlgorithm2N;
              dt, callback = nothing, kwargs...)
```

We don't use the `T` in the code below, do we?

Reply:
Copy that!